keyboards/rura66/rules.mk

```diff
@@ -0,0 +1,40 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader
+BOOTLOADER = caterina
+
+# change yes to no to disable
+#
+# BOOTMAGIC_ENABLE = no   # Virtual DIP switch configuration
+# MOUSEKEY_ENABLE = no    # Mouse keys
+# EXTRAKEY_ENABLE = no    # Audio control and System control
+# CONSOLE_ENABLE = no     # Console for debug
+# COMMAND_ENABLE = no     # Commands for debug and configuration
+# NKRO_ENABLE = no        # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+# BACKLIGHT_ENABLE = yes  # Enable keyboard backlight functionality
+# MIDI_ENABLE = no        # MIDI controls
+# AUDIO_ENABLE = no       # Audio output on port C6
+# UNICODE_ENABLE = no     # Unicode
+# BLUETOOTH_ENABLE = no   # Enable Bluetooth with the Adafruit EZ-Key HID
+# RGBLIGHT_ENABLE = no    # Enable WS2812 RGB underlight.
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+# SLEEP_LED_ENABLE = no   # Breathing sleep LED during USB suspend
+
```

Comment:
```suggestion
# Build Options
```

Reply:
sorry.
I fixed it yesterday and forgot to push it.